scripts/ci/Dockerfile

```diff
@@ -124,7 +129,12 @@ FROM mcr.microsoft.com/playwright:v${PLAYWRIGHT_VERSION}-focal as playwright-bas
 
 FROM playwright-base as output-playwright
 # TODO: remove awscli dependency (157 MB extra)
-RUN apt-get update && apt-get install -y zip awscli && apt-get purge
+RUN apt-get update && apt-get install -y zip && apt-get purge
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
wondering if we should add a separate stage for the aws-cli and then copy the binary from that stage output

```dockerfile
FROM alpine3.15

ARG AWSCLI_VERSION="1.22.24"
ENV AWSCLI_VERSION=${AWSCLI_VERSION}

RUN apk add \
    bash \
    python3 \
    python3-dev \
    py-pip \
    build-base \
    jq \
    curl

RUN pip install awscli==$AWSCLI_VERSION --user
RUN mkdir -p /build

COPY /root/.local/bin/aws /build/
```

```dockerfile
...
COPY --chown=someuser:someuser --from=awscli /build /usr/local/bin
...
```

Reply:
Yep, good idea. Implemented